src/include/rbd/librbd.h

```diff
@@ -1195,6 +1195,15 @@ CEPH_RBD_API ssize_t rbd_aio_compare_and_write(rbd_image_t image,
                                                rbd_completion_t c,
                                                uint64_t *mismatch_off,
                                                int op_flags);
+CEPH_RBD_API ssize_t rbd_aio_comparev_and_writev(rbd_image_t image,
+                                                 uint64_t off, size_t len,
```

Comment:
Why is `len` taken?  I would have expected length to be determined by the iovecs themselves, similar to `rbd_aio_readv` and `rbd_aio_writev`.  If `cmp_len` and `write_len` as determined by `get_iovec_length` don't match, just fail.

Reply:
I wanted the semantics to match with the C++ API, so we can allow iovecs of different sizes (between cmp and write) but if you think that we should not support this then I will remove length.

Reply:
I think consistency of one C API with other C APIs is more preferable, so yeah, let's remove `len`.